src/implementations/utils/dryRun.ts

```diff
@@ -68,9 +68,9 @@ Get more information about gas: https://ethereum.stackexchange.com/questions/3/w
   console.log(green("Information about the transaction:"));
   console.log(
     `Estimated gas required: ${highlight(
-      _estimatedGas.toNumber().toString(10)
+      _estimatedGas.mul(gasPriceAsGwei * gasPriceScale).toNumber()
```

Comment:
why did you change this ? the gas needed depends on the transaction, doesn't dependon the price :thinking:

Reply:
oops. changed wrongly. reverted.